plugin/calcium.lua

```diff
@@ -14,6 +14,9 @@ vim.api.nvim_create_user_command("Calcium", function(opts)
 			mode = "append"
 		elseif args[1] == "replace" or args[1] == "r" then
 			mode = "replace"
+		elseif args[1] == "scratchpad" or args[1] == "s" then
+			require("calcium").floating_scratchpad()
```

Comment:
The reason for `not args[1]` is that this works well as a default mode if nothing is passed after `:Calcium`, just like my other plugin's (Convy.nvim) default mode is also a floating window.

```suggestion
		elseif args[1] == "scratchpad" or not args[1] then
			require("calcium").scratchpad()
```